bindata/linuxptp/ptp-daemon.yaml

```diff
@@ -127,11 +127,11 @@ spec:
             secretName: linuxptp-daemon-secret
         {{ if (eq .EnableEventPublisher true) }}
         - name: pubsubstore
-          {{ if not (hasPrefix "amqp" .EventTransportHost) }}
+          {{ if (eq .TransportStorageType "emptyDir") }}
```

Comment:
Brent told me not to use emptyDir, either CR or PVC, you could fall back to emptydir if the value is empty or CR in a later state.

Reply:
if empty

Reply:
the "emptyDir" is default value set by controller when the value is empty from config

Reply:
updated